Makefile

```diff
@@ -50,8 +45,11 @@ build:
 		$(call check-static-binary,$(BINARY_DEST_DIR)/$$i); \
 	done
 
-docker-build: build
-	docker build --rm -t $(IMAGE) rootfs
+test:
+	go test ./pkg && go test ./pkg/confd && go test ./pkg/env && go test ./pkg/etcd && go test ./pkg/git && go test ./pkg/sshd
```

Comment:
unfortunately neither of those will work, because there are multiple `package main`s in `./pkg/src/` (each builds a small binary that builder uses).

I have a larger refactor in mind that would remove all those small binaries, which would have the side effect of being able to run `go test ./...`, but for now I'd like to leave as-is. Thoughts?